roles/jd-server/src/lib/mempool/mod.rs

```diff
@@ -95,31 +95,34 @@ impl JDsMempool {
     }
 
     pub async fn update_mempool(self_: Arc<Mutex<Self>>) -> Result<(), JdsMempoolError> {
```

Comment:
I suggested a better implementation of this function in this [comment](https://github.com/stratum-mining/stratum/pull/868#issuecomment-2225833093)